### PR TITLE
Localisation crusade 6

### PR DESCRIPTION
**Pull request by arithon**
just some lines to merge for the next update, mostly on german part.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -223,38 +223,38 @@ EVTDESC_WCWAR_312;[Root.PrimaryTitle.GetFOA], I'm giving you [FromFromFrom.GetNa
...
...during slaughter they laugh disgustingly! They call themselves the Laughing Skull clan and look quite mad...\nThough, they do not like the Horde we are familiar with. Perhaps they would be good allies. Their leader, an ogre by the name of Mogor, is willing to help us as a gesture of goodwill. Should we accept their help? The Laughing Skull is quite mad... but perhaps we can control them?;;Das ist erstaunlich. Unter den unz�hligen Ork-Clans gibt es einen, der sich sehr von den anderen unterscheidet. Diese gr�nen Orks, ungeliebt vom Rest ihrer Art, reiten auf Insekten und tragen schaurige Masken aus Chitin. Und w�hrend des Massakers lachen sie auf niedertr�chtige Art und Weise! Sie selbst nennen sich Clan des lachenden Sch�dels und machen einen wahrhaft verr�ckten Eindruck...\nSie sind zwar Orks, �hneln aber nicht der Horde, die wir bereits kennen. Vielleicht w�rden sie gute Verb�ndete abgeben? Ihr Anf�hrer, der Oger Mogor, hat Ner'zhul das Buch des Medivh gestohlen und ist bereit, es uns als Geste des guten Willens zu �berlassen. Aber sollten wir sein Geschenk annehmen? Der Clan des lachenden Sch�dels ist wirklich verr�ckt... aber vielleicht k�nnen wir ihn kontrollieren?;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
EVTDESC_WCBDP_65_alliance;It's amazing. Among the orcish clans, one is found to be very different from the others. These green orcs - unloved by the rest of their kinsmen - ride insects and wear eerie masks of chitin. During their slaughter, they laugh disgustingly! They call themselves the Laughing Skull clan and look quite mad...\nHowever, they do not like the Horde we are familiar with. Perhaps they would be good allies. Their leader, an ogre by the name of Mogor, is willing to help us as a gesture of goodwill. Should we accept their help? They seem utterly insane, but perhaps we can control them?;;Das ist erstaunlich. Unter den unz�hligen Ork-Clans gibt es einen, der sich sehr von den anderen unterscheidet. Diese gr�nen Orks, ungeliebt vom Rest ihrer Art, reiten auf Insekten und tragen schaurige Masken aus Chitin. Und w�hrend des Massakers lachen sie auf niedertr�chtige Art und Weise! Sie selbst nennen sich Clan des lachenden Sch�dels und machen einen wahrhaft verr�ckten Eindruck...\nSie sind zwar Orks, �hneln aber nicht der Horde, die wir bereits kennen. Vielleicht w�rden sie gute Verb�ndete abgeben? Ihr Anf�hrer, der Oger Mogor, hat Ner'zhul das Buch des Medivh gestohlen und ist bereit, es uns als Geste des guten Willens zu �berlassen. Aber sollten wir sein Geschenk annehmen? Der Clan des lachenden Sch�dels ist wirklich verr�ckt... aber vielleicht k�nnen wir ihn kontrollieren?;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -223,38 +223,38 @@ EVTDESC_WCWAR_312;[Root.PrimaryTitle.GetFOA], I'm giving you [FromFromFrom.GetNa
...
+EVTDESC_WCBDP_70;The orcs of the Laughing Skull clan have betrayed us! These bastards were laughing as they cut down our guards and fled through the Dark Portal to Azeroth! Although they helped us the first time, now these fools are on their own, preferring to raid and commit bloody rituals among the peaceful peoples of Azeroth.;;Die Orks aus dem Clan des lachenden Sch�dels haben uns verraten! Diese lachenden Bastarde metzelten unsere Wachen nieder und flohen durch das dunkle Portal nach Azeroth! Obwohl sie uns anfangs geholfen haben, sind diese Verr�ckten jetzt auf sich allein gestellt und ziehen Raubz�ge und blutige Rituale dem friedlichen Leben unter den V�lkern Azeroths vor.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
EVTDESC_WCBDP_70;The orcs of the Laughing Skull clan have betrayed us! These bastards were laughing as they cut down our guards and fled through the Dark Portal to Azeroth! Although they helped us the first time, these fools are now loose on their own, preferring to raid and commit bloody rituals among the peaceful peoples of Azeroth.;;Die Orks aus dem Clan des lachenden Sch�dels haben uns verraten! Diese lachenden Bastarde metzelten unsere Wachen nieder und flohen durch das dunkle Portal nach Azeroth! Obwohl sie uns anfangs geholfen haben, sind diese Verr�ckten jetzt auf sich allein gestellt und ziehen Raubz�ge und blutige Rituale dem friedlichen Leben unter den V�lkern Azeroths vor.;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv (hunk 1 above)
Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv]: done

File: localisation/00_zuwc_united_localisation.csv (hunk 2 above)
Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv]: it is done, although I'm pretty sure, these have been changed only recently in the beyond the dark portal rework...

**Pull request merged**